client/my-sites/checkout/checkout/domain-details-form.jsx

```diff
@@ -387,19 +392,20 @@ export class DomainDetailsForm extends PureComponent {
 	}
 
 	renderCountryDependentAddressFields( needsOnlyGoogleAppsDetails ) {
+		const isInternationalAddress = this.isInternationalAddress();
+
 		return (
 			<div className="checkout__domain-details-country-dependent-address-fields">
-                { ! needsOnlyGoogleAppsDetails && this.renderAddressFields() }
-                { ! needsOnlyGoogleAppsDetails && this.renderCityField() }
-                { ! needsOnlyGoogleAppsDetails && this.renderStateField() }
-                { this.renderPostalCodeField() }
+				{ ! needsOnlyGoogleAppsDetails && this.renderAddressFields() }
+				{ isInternationalAddress && this.renderPostalCodeField() }
+				{ ! needsOnlyGoogleAppsDetails && this.renderCityField() }
+				{ ! isInternationalAddress && ! needsOnlyGoogleAppsDetails && this.renderStateField() }
+				{ ! isInternationalAddress && this.renderPostalCodeField() }
```

Comment:
I wonder if there's a way to `renderPostalCodeField()` once unconditionally rather than once either way.

Reply:
I wondered that too, and thought defining a collection of data structures that would describe the various layouts. And then my brain fell out.

Any ideas for quick wins?